project/dexter-monitor/public/view/createServerDialog.html

```diff
@@ -0,0 +1,41 @@
+<md-dialog layout-padding="" >
+	<md-dialog-content>
+		<div>
+			<h2>Create new Dexter Server</h2>
+			</br>
+			<md-input-container class="md-block" flex-gt-xs="">
```

Comment:
Please create the 'HostIp' label. 
The other overseas used many dexter-server for proiject type.
They need to want input the server ip.